application/src/main/java/com/ericsson/bss/cassandra/ecchronos/application/config/repair/RepairHistoryData.java

```diff
@@ -0,0 +1,212 @@
+/*
+ * Copyright 2024 Telefonaktiebolaget LM Ericsson
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.ericsson.bss.cassandra.ecchronos.application.config.repair;
+
+import com.google.common.base.Preconditions;
+import java.time.Instant;
+import java.util.HashSet;
+import java.util.Set;
+import java.util.UUID;
+
+public final class RepairHistoryData
```

Comment:
I think the current implementation in ecchronos sidecar creates a bean for this table, we do the same for EccNodesSync, I guess it would be good to evaluate how is it done in sidecar, because there's interfaces and some other things, but I'm really fine about having this simple implementation, just put the files in the respective modules

Reply:
This is just information passing object because table has many fields so it is easy to build one object and pass into method. moved to data module.